api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/activation/PReLU.kt

```diff
@@ -36,7 +36,7 @@ public class PReLU(
     public val alphaRegularizer: Regularizer? = null,
     public val sharedAxes: IntArray? = null,
     name: String = ""
-) : AbstractActivationLayer(name) {
+) : AbstractActivationLayer(name), ParametrizedLayer, TrainableLayer {
```

Comment:
But TrainableLayer is a sub-type for Parametrized? Could we remove Parametrized layer from the signature?

Reply:
Done.